integration_tests/testsuite/deploy_app.py

```diff
@@ -72,9 +72,10 @@ def _record_latency_to_bigquery(deploy_latency, language, is_xrt):
     dataset = client.dataset(DATASET_NAME)
     logging.debug('Writing bigquery data to table %s in dataset %s',
                   TABLE_NAME, dataset)
-    table = bigquery.Table(name=TABLE_NAME, dataset=dataset)
-    table.reload()
-    return table.insert_data(row)
+    table_ref = bigquery.TableReference(dataset_ref=dataset,
+                                        table_id=TABLE_NAME)
+    client.get_table(table_ref)
+    return client.create_rows(table_ref, row)
```

Comment:
Just checked the [documentation](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/bigquery/google/cloud/bigquery/client.py#L1023) that if using `table_ref`, then `selected_fields` param is required when calling `create_rows`. So might just passed in a `Table` object:

```
table = client.get_table(table_ref)
return client.create_rows(table, row)
```
I guess you probably wanted to pass in the `Table` object but forgot to create it when calling `client.get_table` :)

Reply:
Ah, yeah that's right. Sorry I thought I had tested this correctly but never actually checked the table I wrote to :fearful: